src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
+    static final String HEADER = "Report for period ";
+    static final String DELIMITER = " ";
+    static final String SEPARATOR = " - ";
+    static final String DOT = "\\.";
+    static final String DASH = "-";
+    static final int ZERO = 0;
+    static final int ONE = 1;
+    static final int TWO = 2;
+    static final int THREE = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String [] dateFromArray = dateFrom.split(DOT);
+        LocalDate dataFromFormatted = LocalDate.parse(dateFromArray[TWO]
```

Comment:
use DateTimeFormatter to create LocalDate like this
![image](https://github.com/mate-academy/jv-salary-info/assets/49767049/f353528e-26b2-40db-a504-9e5a35f980c1)